Cloudinary/Classes/Core/Features/Helpers/CLDTransformation.swift

```diff
@@ -1766,8 +1766,9 @@ import CoreGraphics
         if let rawTrans = params[TransformationParam.RAW_TRANSFORMATION.rawValue], !rawTrans.isEmpty {
             finalComponents.append(rawTrans)
         }
-        
-        return finalComponents.joined(separator: CLDTransformation.transformationContentSeparator)
+                 
+        let joinedComponents = finalComponents.joined(separator: CLDTransformation.transformationContentSeparator)
+        return joinedComponents.replacingOccurrences(of: " ", with: "_")
```

Comment:
This is the bug, not the fix - the spaces in a named transformation should be encoded (`%20`) not turned into underscores.

See the node pr as reference (with a test case to verify) - https://github.com/cloudinary/cloudinary_npm/pull/321/files